tests/listener_test.cc

```diff
@@ -241,6 +245,35 @@ TEST(listener_test, listener_bind_ephemeral_v6_port)
     ASSERT_TRUE(true);
 }
 
+TEST(listener_test, listener_bind_unix_domain)
+{
+    // Avoid name conflict by binding within a fresh temporary directory.
+    using charArray        = char[];
```

Comment:
I'm not sure this type alias makes the code more readable

Reply:
Reverted to C-style array declaration syntax

Reply:
If possible, try std::array.

Reply:
Yes, that's doable.  Changed accordingly.